src/medCore/views/medAbstractView.h

```diff
@@ -61,6 +61,7 @@ class MEDCORE_EXPORT medAbstractView: public dtkAbstractView
     virtual QWidget* toolBarWidget();
 
     virtual QList<medAbstractNavigator*> navigators();
+    virtual QList<medAbstractNavigator*> extraNavigators();
     virtual QList<medAbstractInteractor*> interactors();
 
```

Comment:
Why did you need to have extra navigators specifically? isn't navigators enough ? If you really need it, I would be more in favour to put all the 'primary /extra navigators / interactors' accessor public for consistency.